.github/workflows/ci.yaml

```diff
@@ -5,10 +5,10 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ '12', '14', '16' ]
+        node-v: [16.x, 18.x, 20.x]
```

Comment:
Current node version is 24. Refer https://nodejs.org/en/about/previous-releases 
Add Node 22, 24 versions